### ISSUE TITLE
Very slow startup when packages aren't yet cached

### ISSUE THREAD

**Issue by non-Jedi** [julia-vscode/SymbolServer.jl#56]
As an example, I'm still waiting for the symbol server store to be set with https://github.com/non-Jedi/ChemicalEngineeringToolbox.jl/blob/master/Project.toml after something like 15 minutes. I'm assuming that generating the ".jstore" files is just a very slow process? It doesn't seem to be jit startup time because I'm getting every couple minutes a new `[ Info: Interpolations not stored on disc`, so it must be that the operation is just slow no matter what.

**Comment by davidanthoff** [comment]
So we're going to tackle this kind of problem after we release v0.13 of the VS Code extension.

**Comment by phlavenk** [comment]
I'd like to ask, the --uuid--.jstore are cached for unique package. How it distinguishes between different versions of the package, if more of them are installed and, e.g, used simultaneously in 2 different instances of VScode? Should it not also include similar strategy as Julia 1.3 with keeping .ji files for last ~20 versions of a package? I'd be glad for clarification.

**Comment by davidanthoff** [comment]
Yeah, we might need a much more fine grained story there... Maybe also differentiate based on what Julia version was used.

**Comment by davidanthoff** [comment]
`master` now has a version that does the whole indexing async in the background. That should unblock a lot of things, in particular the LS should start working long before the indexing is done.

We should still try to improve performance of the indexing.

We also now store the cache per version of the package, no longer by UUID. The cache is essentially now stored by the git tree hash (with some twists for deved and stdlib packages).
